staging.Dockerfile

```diff
@@ -0,0 +1,74 @@
+# This Docker file tends to be used only
+# for `development` and `test` stage
+#
+# CLI command to run this file: script="some-name" docker-compose up --build
+# `some-name` is one of the names provided under `scripts` tag in `package.json` file
+# example: $ script="test" docker-compose up --build    --> will run telescope with `test` script
+# default: $ docker-compose up --build                  --> will run telescope with `start` script
+
+
+# Dockerfile
+#
+# ------------------
+# Context: Base Container
+FROM node:12.15.0-alpine3.9 AS base
```

Comment:
Question: why switch from `FROM node:lts` as you do above, and not use `FROM node:lts-alpine`?

Reply:
Node:LTS is based off of Debian Stretch https://github.com/nodejs/docker-node/blob/46f5203674c748b0701135c8eeea4b250b3ecb6d/12/stretch/Dockerfile which is bigger, great for ensuring we are able to install / compile dependencies before moving the final artifacts to the smaller Alpine container.

Reply:
I misunderstood what you meant before, looking now I see a better implementation which addresses your comment. Testing now.